source/requests/streams.d

```diff
@@ -634,122 +634,133 @@ public unittest {
     assert(c.length == c_length);
 }
 
-extern(C) {
-    int SSL_library_init();
-    void OpenSSL_add_all_ciphers();
-    void OpenSSL_add_all_digests();
-    void SSL_load_error_strings();
-    
-    struct SSL {}
-    struct SSL_CTX {}
-    struct SSL_METHOD {}
-    
-    SSL_CTX* SSL_CTX_new(const SSL_METHOD* method);
-    SSL* SSL_new(SSL_CTX*);
-    int SSL_set_fd(SSL*, int);
-    int SSL_connect(SSL*);
-    int SSL_write(SSL*, const void*, int);
-    int SSL_read(SSL*, void*, int);
-    int SSL_shutdown(SSL*) @trusted @nogc nothrow;
-    void SSL_free(SSL*);
-    void SSL_CTX_free(SSL_CTX*);
-    
-    long    SSL_CTX_ctrl(SSL_CTX *ctx, int cmd, long larg, void *parg);
-    
-    long SSL_CTX_set_mode(SSL_CTX *ctx, long mode);
-    long SSL_set_mode(SSL *ssl, long mode);
-    
-    long SSL_CTX_get_mode(SSL_CTX *ctx);
-    long SSL_get_mode(SSL *ssl);
-    
-    SSL_METHOD* SSLv3_client_method();
-    SSL_METHOD* TLSv1_2_client_method();
-    SSL_METHOD* TLSv1_client_method();
-}
+version (sslLibs) {
```

Comment:
Hello,

Do we really need "version(ssLibs)"?
Its presence require adding -version=sslLibs to each dmd call when we do not use dub:
```
igor$ cat y.d
#!/usr/bin/env rdmd -I./source librequests.a -L-lssl -L-lcrypto y.d
import std.stdio;
import std.algorithm;
import requests;
import std.experimental.logger;

void main()
{
    globalLogLevel(LogLevel.trace);
    Request rq;
    rq.verbosity = 3;
    rq.useStreaming = true;
    auto rs = rq.get("http://httpbin.org/stream/100");
    auto stream = rs.receiveAsRange;
    writeln(stream.joiner.count);
}
igor$ ./y.d
./source/requests/streams.d(1005): Error: undefined identifier 'SSLSocketStream'
Failed: ["dmd", "-I./source", "librequests.a", "-L-lssl", "-L-lcrypto", "-v", "-o-", "y.d", "-I."]
```
From the other side removing it makes no harm for vibed tests.
So can i remove it?